docker/complement/conf/workers-shared-extra.yaml.j2

```
@@ -88,3 +94,13 @@ server_notices:
   system_mxid_display_name: "Server Alert"
   system_mxid_avatar_url: ""
   room_name: "Server Alert"
+
+{#
+  This file is intended to extend the configuration with Complement-specific
+  tweaks, so here we include the original section of configuration that does
+  not have any Complement-specific values.
+
+  The base configuration is moved out of the default path to `shared-orig.yaml.j2`
+  in the Complement Dockerfile.
+#}
```

please can you move this to the top of the file?

```suggestion
{#
  This file extends the default "shared" configuration file (from the "synapse-workers"
  docker image) with Complement-specific  tweak.
  
  The base configuration is moved out of the default path to `shared-orig.yaml.j2`
  in the Complement Dockerfile, and below we include that original file.
#}
```